Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingController.swift

```diff
@@ -3,6 +3,7 @@
 
 import SwiftUI
 
+#if !os(macOS)
```

Comment:
should this also be `#if canImport(UIKIt)` for consistency? I think it would have the same effect here

Reply:
I think this is fine either way... I guess `canImport` is weird if you aren't actually importing the thing you are asking about so maybe what you have is best.

Reply:
I lean a bit towards using `#if os(macOS)` over `#if canImport(AppKit)` etc but dont have much of an opinion since they mean the same thing